translations/README-he.md

```diff
@@ -53,18 +52,26 @@
     </p>
     <hr />
 </div>
+<div dir='rtl'>
+	<h2 name='what-is-it'> על מה מדובר כאן?</h2>
```

Comment:
I'd rather stick to github markup instead of using straight HTML to make editing/updating easier for everyone.